src/index.js

```diff
@@ -0,0 +1,86 @@
+const fs = require("fs");
+const inquirer = require("inquirer");
+const figlet = require("figlet");
+const path = require("path");
+const open = require("open");
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const {
+  managerQuestions,
+  confirmNextStep,
+  engineerQuestions,
+  internQuestions,
+} = require("./questions");
+const { generateHTML } = require("./generate");
+
+const init = async () => {
+  let inProgress = true;
+
+  const engineerInfo = [];
+  const internInfo = [];
+
+  const managerAnswers = await inquirer.prompt(managerQuestions);
+
+  const manager = new Manager(
+    managerAnswers.name,
+    managerAnswers.id,
+    managerAnswers.email,
+    managerAnswers.officeNumber,
+    managerAnswers.teamName
+  );
+
+  while (inProgress) {
+    const { nextStep } = await inquirer.prompt(confirmNextStep);
+
+    if (nextStep === "engineer") {
+      const engineerAnswers = await inquirer.prompt(engineerQuestions);
+
+      const engineer = new Engineer(
```

Comment:
In this case I would probably suggest that we set this class up to expect an object with the 4 keys, rather than 4 separate arguments as we have here. When we have a function or class which is expecting multiple arguments (generally anything more than 2), it carries a higher risk of error, as we must always ensure to pass the arguments in the specific order they were set up.